src/resources/js/aggregator-fields.js

```diff
@@ -106,7 +105,7 @@ tribe_aggregator.fields = {
 				obj.reset_preview()
 
 				// Every time you change Type of import we reset the frequency field
-				var $this = $( this ),
+				var	$this = $( this ),
```

Comment:
Shouldn't it be done with spaces though? I feel like our coding guidelines say as much (ie, initial indent with tabs, subsequent alignment with spaces):

```
> > > var $something,
> > >     $other;
```

(Where `>` is a tab)